pyproject.toml

```diff
@@ -1,12 +1,12 @@
 [tool.poetry]
 name = "bechdelai"
-version = "0.0.1-alpha.2"
+version = "0.1.0"
 description = "Automating the Bechdel test and its variants for feminine representation in movies with AI"
-authors = ["Data For Good <hellodataforgood@gmail.com>"]
+authors = ["Théo Alves Da Costa <theo.alves.da.costa@gmail.com>"]
```

Comment:
Ahah, non je préférais quand c'était bien Data For Good l'auteur pourquoi ça revient ici ?

Reply:
Aucune idée.

Reply:
Ah oui c'est ça qui date d'il y a un an, pas grave, on va bien remettre :)